pkg/util/azureclient/mgmt/storage/blobcontainers.go

```diff
@@ -0,0 +1,31 @@
+package storage
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
+	"github.com/Azure/go-autorest/autorest"
+
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+)
+
+type BlobContainersClient interface {
```

Comment:
Is this being used? I couldn't find any references to it. Could it be removed?

Reply:
Thanks for pointing, it is removed now.